index.html

```diff
@@ -36,6 +36,10 @@
         };
 
         let libView = new LibraryEntryPoint.LibraryView(configuration);
+
+        libView.on("itemClicked", function(item) {
+            console.log(item.props.data.text);
+        })
```

Comment:
The client doesn't care about the props, they just need the string. This is incorrect. Just do `console.log(item)` and update the code below. No one else outside of `librarie.js` will know the internals of `item.props.data`...